sway-fmt-v2/src/utils/punctuated.rs

```diff
@@ -4,9 +4,13 @@ use crate::{
     FormatterError,
 };
 use std::fmt::Write;
-use sway_ast::{keywords::CommaToken, punctuated::Punctuated, StorageField, TypeField};
+use sway_ast::{
+    keywords::CommaToken, punctuated::Punctuated, token::PunctKind, StorageField, TypeField,
+};
 use sway_types::{Ident, Spanned};
 
```

Comment:
```suggestion
```

In general I think it's good to keep `use` statements together as it allows rustfmt to automatically order them, but not that important.